tests/client_tests/python/asyncpg/test_asyncpg.py

```diff
@@ -4,45 +4,54 @@
 from crate.qa.tests import NodeProvider
 
 
-async def exec_queries(test, hosts):
-    pool = await asyncpg.create_pool(f'postgres://crate@{hosts}/doc')
-    async with pool.acquire() as conn:
-        result = await conn.execute(
-            "CREATE TABLE t1 (x int primary key, y int)")
-        test.assertEqual(result, 'CREATE 1')
+async def basic_queries(test, conn):
+    """
+    Invoke some basic queries.
+    """
```

Comment:
```suggestion
```

Doesn't really add any meaningful information

Reply:
I've remove this comment.